cmd/maya-apiserver/app/command/start.go

```diff
@@ -126,7 +127,12 @@ func Run(cmd *cobra.Command, c *CmdStartOptions) error {
 	if mconfig == nil {
 		return errors.New("Unable to load the configuration.")
 	}
-
+	go func(){
+		err := spc.Start()
+		if err != nil{
+			glog.Errorf("coould not start controller: %s", err.Error())
```

Comment:
Logs should start with upper case. Typo in "could".

Reply:
If these logs will be a part of maya-api-server, it may be better to have the module name embedded in the message. This may have to be done as a part of logging setup.

Reply:
Sure.